<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerForm</class>
 <widget class="QWidget" name="ControllerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Distance kernel: L2 norm&lt;/span&gt;&lt;br&gt;&lt;span style=&quot; font-size:7pt; font-style:italic;&quot;&gt;(future extensions placeholder)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,3">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox { border: 1px solid gray;border-radius: 2px;margin-top: 0.5em; }; QGroupBox::title{subcontrol-origin: margin;subcontrol-position: top left;padding: 5 5px;font-weight: bold;}</string>
        </property>
        <property name="title">
         <string># of sampled distances</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,4,2">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxSampleDists">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>10000</number>
           </property>
           <property name="maximum">
            <number>1000000000</number>
           </property>
           <property name="singleStep">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbExhaustiveDists">
           <property name="text">
            <string>All</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="newClusterBtn">
           <property name="text">
            <string>New cluster</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>268</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">padding:0</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="clusterSupport">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>172</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding:1</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
